drivers/gke/gke_driver.go

```diff
@@ -132,9 +143,10 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 	driverFlag.Options["zone"] = &types.Flag{
 		Type:  types.StringType,
 		Usage: "The zone to launch the cluster",
-		Default: &types.Default{
```

Comment:
is there a different way you can do this or at least fill the default at some point?

Reply:
Not unless I can differentiate between "zone is set to the actual default" vs "zone is explicitly set to a value which happens to match the default".

I'm not sure there's that much value in the default other, unless there's some data to backup that kontainer-engine/Rancher users overwhelmingly provision clusters in us-east1-a?

Reply:
I could fill the default again in `validate()` if region is empty? That feels very dirty though, and potentially surprising.

Reply:
One alternative that came to mind. Instead of introducing a `region` attribute we can call it `location` instead, and leave the zone with it's default.

The rule then goes:
* if location is empty, get the value from zone (which will retain its default)
* if location is not empty, zone gets ignored

This allows for moving to the more generic "location" term, which can be one of zone or region (in the same way the GKE SDK uses location) while not breaking the current usage of zone and its default.

Reply:
the last option may work, the default may not be a big deal, it just depends on if it is being used.